test/byContract/IexecEscrow/IexecEscrowToken.test.ts

```diff
@@ -0,0 +1,473 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { AddressZero } from '@ethersproject/constants';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber } from 'ethers';
+import { ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    IexecInterfaceToken,
+    IexecInterfaceToken__factory,
+    RLC,
+    RLC__factory,
+} from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+const standardAmount = AmountWithDecimals(BigNumber.from(100));
+
+describe('IexecEscrowToken', () => {
```

Comment:
Maybe it worth adding totalSupply checks when mint or burn is called

Reply:
[add totalSupply checks on tests](https://github.com/iExecBlockchainComputing/PoCo/pull/141/commits/580677250ce6b433e82d747e7d40ea0f9b10c270)